src/app/containers/Metadata/index.jsx

```diff
@@ -83,6 +84,7 @@ const MetadataContainer = ({
   const htmlAttributes = {
     dir,
     lang,
+    className: onClient() ? 'js' : 'no-js',
```

Comment:
good spot @jroebu14, just checked [helmet tests](https://github.com/nfl/react-helmet/blob/master/test/HelmetTest.js#L417) and it's been used as `class` instead of `className` either way react still maps it as `class` on the browser. will refactor to use `class`